configurations/init_test_workspace.py

```diff
@@ -0,0 +1,53 @@
+import json
+
+from core_data_modules.cleaners import swahili
+from core_data_modules.data_models import CodeScheme
+from temba_client.v2 import Field
+
+from src.common.configuration import RapidProClientConfiguration, CodaClientConfiguration, UUIDTableClientConfiguration, \
+    EngagementDatabaseClientConfiguration
+from src.engagement_db_to_coda.configuration import CodaSyncConfiguration, CodaDatasetConfiguration, \
+    CodeSchemeConfiguration
+from src.engagement_db_to_rapid_pro.configuration import EngagementDBToRapidProConfiguration, DatasetConfiguration, \
+    WriteModes, ContactField
```

Comment:
consider using tuples to do multi-line imports
![Screenshot 2021-06-14 at 15 12 36](https://user-images.githubusercontent.com/39700595/121890429-023c7c80-cd23-11eb-8c81-8542f42d1a38.png)